src/store/store.ts

```diff
@@ -62,6 +72,70 @@ export const deleteStatus = (state: RootState, action: DeleteStatus) => {
       });
     });
   });
+
+  state.workflows.forEach((workflow: IWorkflow) => {
+    workflow.transitions.forEach((value, key) => {
+      if (key.id === statusId) {
+        workflow.transitions.delete(key);
+      }
+
+      value.forEach((status, index) => {
+        if (status.id === statusId) {
+          value.splice(index, 1);
+        }
+      })
+    });
+  });
+};
+
+export const createWorkflow = (state: RootState, action: CreateWorkflow) => {
+  const { name, transitions } = action.payload;
+
+  const correctTransitions = new Map();
+  state.statuses.forEach((status, index) => {
+    const transition = transitions[index] ? transitions[index] : [];
```

Comment:
Why do we use the status index for the transitions index?

Reply:
done